test/conf_test.py

```diff
@@ -17,16 +17,11 @@
 sys.argv.extend([
     '--config-value=flag_key=flag_value',
     '--config-value', 'other_flag=other_value',
-# You can specify arbitrary keys, but they'll get ignored if they aren't
```

Comment:
Because this was messing with the actual sys.argv used in other tests. I added a tearDownModule to fix it